.github/workflows/publish_docker_runner.yml

```diff
@@ -26,3 +26,28 @@ jobs:
       if: github.ref != 'refs/heads/main'
       run: bash ./starter_scripts/push_docker_runner_image.sh "ci-release-test"
 
+  publish-test-container:
+    needs: publish-main
+
+    runs-on: ubuntu-latest
+    environment: publish
+      - uses: actions/checkout@v2
+      - name: Build TestContainer
+        run: ./start-test-env build-test-container
```

Comment:
Not sure if I actually need that. `push-test-container` should build automatically.

Reply:
Does ./start-test-env build-test-container use rebuild? Should we force rebuild? Maybe for updates.

Reply:
both `push-test-container` and `build-test-container` support --force-rebuild